assets/components/legal/legalSection/legalSectionContainer.js

```diff
@@ -4,21 +4,14 @@
 
 import { connect } from 'react-redux';
 
-import type { CommonState } from 'helpers/page/commonReducer';
-
 import LegalSection from './legalSection';
 
-
 // ----- State Maps ----- //
 
-function mapStateToProps(state: { common: CommonState }) {
-
-  return {
-    countryGroupId: state.common.internationalisation.countryGroupId,
-  };
-
-}
-
+const mapStateToProps = state => ({
+  countryGroupId: state.common.internationalisation.countryGroupId,
+  contributionType: state.page.form ? state.page.form.contributionType : '',
```

Comment:
the old payment flow doesn't seem to have `state.page.form` so I had to put this check in to let it render, this means the updated T&C will always show.